### PR TITLE
Add display of resource policies name in describe backup output

### PR DESCRIPTION
**Pull request by qiuming-best**
Thank you for contributing to Velero!

# Please add a summary of your change

# Does your change fix a particular issue?

Fixes #(issue)
https://github.com/vmware-tanzu/velero/issues/6066
# Please indicate you've done the following:

- [ ] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [ ] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required` as a comment on this pull request.
- [ ] Updated the corresponding documentation in `site/content/docs/main`.

### CODE REVIEW

File: pkg/cmd/util/output/backup_describer.go [hunk 1]
```diff
@@ -83,6 +83,11 @@ func DescribeBackup(
 
 		d.Printf("Phase:\t%s%s\n", phaseString, logsNote)
 
+		if backup.Spec.ResourcePolicy != nil {
+			d.Println()
+			d.Printf("Resource policies:\t%s\n", backup.Spec.ResourcePolicy.Name)
```
Comment (shubham-pampattiwar) [inline pkg/cmd/util/output/backup_describer.go:88]: Can a backup be associated with multiple resource policies ?

Comment (qiuming-best) [reply pkg/cmd/util/output/backup_describer.go:88]: no, but one resource policies could be associated with multiple backup.

Comment (reasonerjt) [reply pkg/cmd/util/output/backup_describer.go:88]: This is the name of the configmap right?  
IMO it would be better to also output the reference type.

Comment (qiuming-best) [reply pkg/cmd/util/output/backup_describer.go]: updated

**Review by ywk253100 — APPROVED** [review]

**Pull request merged**